cmd/qat_plugin/qat_plugin.go

```diff
@@ -208,6 +211,25 @@ func isValidVfDeviceID(vfDevID string) bool {
 	return false
 }
 
+func swapBDF(devstrings []string) (result []string) {
+	for _, dev := range devstrings {
+		tmp := strings.Split(dev, ":")
+		result = append(result,
+			fmt.Sprintf("%v:%v:%v", tmp[2], tmp[1], tmp[0]))
+	}
+	return
+}
+
+func sortByFunction(bdf []string) []string {
+
+	// make it "FDB" and string sort
+	tmp := swapBDF(bdf)
+	sort.Strings(tmp)
+
+	// return "BDF"
+	return swapBDF(tmp)
+}
```

Comment:
`sortByFunction()` should have a unit test.

Reply:
Done.